src/sas/qtgui/Plotting/Slicers/AnnulusSlicer.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, base, axes, item=None, color='black', zorder=3):
         self.connect = self.base.connect
 
         # Number of points on the plot
-        self.nbins = 36
+        self.nbins = 100
```

Comment:
Beyond my skill level at this point but if it would make the code better/cleaner for the future happy to be educated .. or have someone else do it properly.  I assume we can merge this and then discuss if it is worth doing the above and if so how?